app/views/users/show.html.erb

```diff
@@ -1 +1,25 @@
-<h1>Here are the detials about User</h1>
+<div class="m-4" style="display:flex; flex-direction:column; align-items:stretch; row-gap:2vh">
+<section class="rounded-lg p-2 flex items-center">
+    <img class="border" src="<%= @user.photo %>">
+    <div class="ml-4 text-center flex inform border">
+      <h2 class="font-bold mr-4 mb-2 header-font"><%= @user.name %></h2>
+      <p class="text-red-600 par">Number of posts: <%= @user.posts_counter %></p>
+    </div>
+    <!-- display user's posts, comments, likes, etc. -->
+</section>
+<section class="rounded-lg border p-8 items-center">
+        <h3 class="font-bold">Bio</h3>
+        <p class="mt-4">
+        <%= @user.bio %>
+        </p>
+</section>
+<% @user.recent_posts.each do |post| %>
+<section class="rounded-lg p-8 items-center border"">
+    <h3 class="font-bold">Post #<%= post.id %></h3>
+        <p class="mt-4"><%= post.text %></p>

```

Comment:
- [ ] Please currently the comments box doesn't match the design , please fix it ,the comments should be in a one box I see that each is in different box in your code
 
| your Design | original|
| ------------------ | ---------- |
| ![image](https://github.com/mohisa302/Blog-app/assets/74744787/3433554b-573e-4e5d-bbd0-cd3607b854b3) | ![image](https://github.com/mohisa302/Blog-app/assets/74744787/6bd26dda-d759-49f5-b569-d05386c7d911) |